api/db/repositories/customerRepository.js

```diff
@@ -1,25 +1,52 @@
-const Customers = require('@db/dbHelper').Customers;
+const { Customers } = require('@db/dbHelper')
+const { Sequelize } = require('@db/dbHelper')
 
-exports.Customers = Customers;
+const Op = Sequelize.Op
+exports.Customers = Customers
 
 exports.createCustomer = async (customerData) => {
```

Comment:
remove async

Reply:
in the whole file where there are no awaits going on

Reply:
done